src/lib/profiles/data-management/Current/LoggingManagement.cpp

```diff
@@ -400,52 +400,29 @@ void LoggingManagement::SkipEvent(EventLoadOutContext * aContext)
 }
 
 /**
- * @brief Create and initialize the logging management buffers. Must
- *   be called prior to the logging being used.
+ * @brief Create LoggingManagement object and initialize the logging management
+ *   subsystem with provided resources.
  *
- * @param[in] inMgr         WeaveExchangeManager to be used with this logging subsystem
- *
- * @param[in] inNumBuffers  Number of buffers to use for event storage
- *
- * @param[in] inBufferLengths Description of inBufferLengths
- *
- * @param[in] inBuffers     The buffers to use for actual event logging.
- * @param[in] inCounterKeys Keys naming persisted counters
- *
- * @param[in] inCounterEpochs An array of epochs for each of the persisted counters
- *
- * @param[in] inCounterStorage Application-provided storage for the persistent counters
- */
-void LoggingManagement::CreateLoggingManagement(nl::Weave::WeaveExchangeManager * inMgr, size_t inNumBuffers,
-                                                size_t * inBufferLengths, void ** inBuffers,
-                                                nl::Weave::Platform::PersistedStorage::Key * inCounterKeys,
-                                                const uint32_t * inCounterEpochs, nl::Weave::PersistedCounter ** inCounterStorage)
-{
-    new (&sInstance)
-        LoggingManagement(inMgr, inNumBuffers, inBufferLengths, inBuffers, inCounterKeys, inCounterEpochs, inCounterStorage);
-}
-
-/**
- * @brief Create and initialize the logging management buffers. Must
- *   be called prior to the logging being used.
- *
- * @param[in] inMgr         WeaveExchangeManager to be used with this logging subsystem.
+ * Initialize the LoggingManagement with an array of LogStorageResources.  The
+ * array must provide a resource for each valid importance level, the elements
```

Comment:
The function description should be updated because elements order is not important in current implementation.

Reply:
Order remains important, and the function expects the elements to be ordered.  By providing the importance, we admit to gaps and non-contiguous importance enums.  Documentation is correct.